code/game/gamemodes/antag_spawner.dm

```diff
@@ -205,7 +205,36 @@
 
 
 ///////////SLAUGHTER DEMON
+/obj/item/weapon/antag_spawner/slaughter_demon
+	name = "Bottle of Blood"
+	desc = "A bottle of magically infused blood, the smell of which will attract extradimensional beings when broken."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "vial"
+	
+/obj/item/weapon/antag_spawner/slaughter_demon/proc/check_usability(mob/user)
+	if(used)
+		user << "<span class='warning'>Somehow the bottle has already been emptied!</span>" // shouldn't happen as it destroys itself upon succesful summoning
+		return 0
+	if(user.z == ZLEVEL_CENTCOM)
+		user << "<span class='warning'>There are no demons within this realm. Try using the [src] onboard the station.<span>"
+		return
+	return 1
 
+/obj/item/weapon/antag_spawner/slaughter_demon/attack_self(mob/user)
+	if(!(check_usability(user)))
```

Comment:
You should not use ! here, otherwise it'll do the exact opposite aka fail when you can use it, and succeed when you can't.